src/Database/Database.php

```diff
@@ -1006,6 +1006,33 @@ public function insert($table, array $param, bool $on_duplicate_update = false)
 		return $this->e($sql, $param);
 	}
 
+	/**
+	 * Replace a row of a table
```

Comment:
Please be more specific, something like
```suggestion
	 * Inserts a row with the provided data in the provided table.
	 * If the data corresponds to an existing row through a UNIQUE or PRIMARY index constraints, it updates the row instead.
```